docs/comparison/traditional_operators.md

```diff
@@ -0,0 +1,95 @@
+2. Fetch the details for the two top stories using the Hacker News API.
+3. Summarize the selected stories Markdown, using Python.
+
+We will implement the first two steps using `BashOperator` and the third step using `PythonOperator`.
+The last task will generate a `Markdown` snippet similar to:
+
+```
+| title                                                                       | url                                                                                                                    |
+|:----------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------|
+| I keep turning my Google Sheets into phone-friendly webapps                 | https://arstechnica.com/gadgets/2024/12/making-tiny-no-code-webapps-out-of-spreadsheets-is-a-weirdly-fulfilling-hobby/ |
+| Coconut by Meta AI – Better LLM Reasoning with Chain of Continuous Thought? | https://aipapersacademy.com/chain-of-continuous-thought/                                                               |
+```
+
+The main logic is implemented as plain Python functions in [hacker_news.py](https://github.com/astronomer/dag-factory/blob/main/dev/dags/hacker_news.py):
+
+```title="pypi_stats.py"
+--8<-- "dev/dags/hacker_news.py:hacker_news"
+```
+
+
+
+## Implementation
+
+As a reference, the following workflows run using Airflow 2.10.2 and DAG Factory 0.21.0.
+
+### Plain Airflow Python DAG
+
+```title="example_hackernews_plain_airflow.py"
+--8<-- "dev/dags/comparison/example_hackernews_plain_airflow.py"
+```
+
+### Alternative DAG Factory YAML
+
+```title="example_hackernews_dagfactory.py"
+--8<-- "dev/dags/comparison/example_hackernews_dagfactory.yml"
+```
+
+
+## Comparison
+
+### Goal
+
+Both implementations accomplish the same goal and result in the expected Markdown table.
+
+### Airflow Graph view
+
+As it can be seen in the screenshots below, both the DAG created using Python with standard Airflow and the
+DAG created using YAML and DAG Factory look identical, from a graph topology perspective, and also from the underlining
+operators being used.
+
+#### Graph view: Plain Airflow Python DAG
+
+![alt text](../static/example_hackernews_plain_airflow_graph.png "Python DAG Graph visualisation")
+
+#### Graph view: Alternative DAG Factory YAML
+
+![alt text](../static/example_hackernews_dagfactory_graph.png "YAML DAG Graph visualization")
+
+### Airflow Code view
+
+From an Airflow UI perspective, the content displayed in the "Code" view is the main difference between the two implementations. While Airflow renders the original Python DAG, as expected, in the case of the YAML DAGs, Airflow displays the Python file that references the DAG Factory YAML files:
+
+```title="example_load_yaml_dags.py"
+--8<-- "dev/dags/example_load_yaml_dags.py"
+```
```

Comment:
```suggestion
```

Looking at the built docs, this block seems redundant to me as the image in the below `Code View: Alternative DAG Factory YAML` shows the same snippet.

Reply:
Just replied in: https://github.com/astronomer/dag-factory/pull/327/files#r1901720084

Once we agree on the approach, I'll fix it in both places!